scripts/rv_expression_association/run_SKAT.R

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env Rscript
+
+suppressPackageStartupMessages(suppressMessages(install.packages("googleCloudStorageR", repos = 'http://cran.csiro.au', quietly = T)))
+suppressPackageStartupMessages(suppressMessages(install.packages("SKAT", repos = 'http://cran.csiro.au/', quietly = T)))
+
+suppressPackageStartupMessages(library(googleCloudStorageR, quietly = T))
+suppressPackageStartupMessages(library(gargle, quietly = T))
+suppressPackageStartupMessages(library(dplyr, quietly = T))
+suppressPackageStartupMessages(library(SKAT, quietly = T))
+
+# token authorisation
+scope <- c("https://www.googleapis.com/auth/cloud-platform")
+token <- token_fetch(scopes = scope)
+gcs_auth(token = token)
+
+# set bucket
+googleCloudStorageR::gcs_global_bucket("gs://cpg-tob-wgs-test")
+
+# sample mapping file, matching CPG IDs to OneK1K IDs
+smf <- googleCloudStorageR::gcs_get_object("scrna-seq/grch38_association_files/OneK1K_CPG_IDs.tsv")
+smf_df <- as.data.frame(smf)
+colnames(smf_df)[2] <- "CPG_ID"  # change column name to match other files
+
+# expression file (gene: VPREB3, cell type: naive B cells)
+y <- googleCloudStorageR::gcs_get_object("v0/vpreb3_B_naive_expression.csv")
```

Comment:
Just a general nit comment, but although I think these are the variable names SKAT uses, they're quite hard to follow

Reply:
ok, yes as you said I was trying to stick with the SKAT nomenclature but will try to give more informative names! As a general comment while I use R a lot for plotting I have not really written scripts like this very often (vs Python), so apologies for the poor form